qmi/instruments/newport/smc_100cc.py

```diff
@@ -30,3 +44,542 @@ def __init__(self,
+                                it is set to the initialised value of the controller address.
+        """
+        _logger.info(
+            "Getting the max. output voltage of the driver to the motor of instrument [%s]", self._name)
+        self.controller_address = controller_address
+        driver_voltage = self._scpi_protocol.ask(
+            self._build_command("DV?"))
+
+        self._check_error()
+        return float(driver_voltage[3:])
+
+    @rpc_method
+    def set_driver_voltage(self, driver_voltage: float, controller_address: Optional[int] = None) -> None:
+        """
+        Set the max. output voltage of the driver to the motor.
+
+        Parameters:
+            driver_voltage:      New profile generator base velocity.
+            controller_address: Optional address of the controller that needs to be controlled. By default,
+                                it is set to the initialised value of the controller address.
+        """
+        if 12.0 > driver_voltage or driver_voltage > 48.0:
+            raise QMI_InstrumentException(
+                f"Provided value {driver_voltage} not in valid range 12 >= driver_voltage >= 48.")
+
+        self.controller_address = controller_address
+        _logger.info(
+            "Setting the max. output voltage of the driver to the motor of instrument [%s] to [%f]",
+            self._name, driver_voltage
+        )
+        self._scpi_protocol.write(self._build_command(
+            "DV", driver_voltage))
+        self._check_error()
+
+    @rpc_method
+    def get_low_pass_filter_cutoff_frequency(self, controller_address: Optional[int] = None) -> float:
+        """
+        Get the low pass filter cut-off frequency Kd.
+
+        Parameters:
+            controller_address: Optional address of the controller that needs to be controlled. By default,
+                                it is set to the initialised value of the controller address.
+
+        Returns:
+            The low pass filter cut-off frequency, Kd, in Hertz.
+        """
+        _logger.info(
+            "Getting encoder increment value of instrument [%s]", self._name)
+        # instrument must be in configuration state to get the encoder increment value.
+        self.reset(controller_address)
+        self.enter_configuration_state(controller_address)
+        res = self._scpi_protocol.ask(
+            self._build_command("FD?"))
+        sleep(self.COMMAND_EXEC_TIME)
+        self._check_error()
+        self.exit_configuration_state(controller_address)
+        return float(res[3:])
+
+    @rpc_method
+    def set_low_pass_filter_cutoff_frequency(
+            self, frequency: float, persist: bool = False, controller_address: Optional[int] = None) -> None:
+        """
+        Set the low pass filter cut-off frequency Kd.
+
+        Parameters:
+            frequency:          Cutoff frequency Kd in Hertz.
```

Comment:
if values have specific ranges I'm partial to also mentioning this in the docstring. Also applies to other instances

Reply:
This I've added to docstrings where applicable (please check if I did them all).